lib/facter/swap_file_size.rb

```diff
@@ -0,0 +1,19 @@
+Facter.add('swap_file_size') do
```

Comment:
Might be an idea to change this to a "swap_file_%{name}_size" because it's possible to have multiple swap files, so you'd want to check the size of the named swap file